From 72c02387f6e1a9d2791dd78533d0c3e909400715 Mon Sep 17 00:00:00 2001
From: Sitaram Chamarty <sitaram@atc.tcs.com>
Date: Fri, 25 Nov 2016 18:16:33 +0530
Subject: [PATCH] fix shebang in repo-specific-hooks

(thanks to Dieter on the mailing list for catching this!)
---
 src/triggers/repo-specific-hooks | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/triggers/repo-specific-hooks b/src/triggers/repo-specific-hooks
index 7ed28e5..6e229ea 100755
--- a/src/triggers/repo-specific-hooks
+++ b/src/triggers/repo-specific-hooks
@@ -85,7 +85,7 @@ for my $repo (keys %repo_hooks) {
 }
 
 __DATA__
-#/bin/sh
+#!/bin/sh
 
 # Determine what input the hook needs
 # post-update takes args, pre/post-receive take stdin
